pom.client.xml

```diff
@@ -115,14 +115,14 @@
     <groovy-eclipse-batch.version>2.4.16-03</groovy-eclipse-batch.version>
     <client-runtime.version.v2>2.1.1</client-runtime.version.v2>
     <slf4j-api.version>1.7.5</slf4j-api.version>
-    <azure-common.version>1.0.0-SNAPSHOT</azure-common.version>
-    <azure-common-auth.version>1.0.0-SNAPSHOT</azure-common-auth.version>
-    <azure-common-mgmt.version>1.0.0-SNAPSHOT</azure-common-mgmt.version>
+    <azure-core.version>1.0.0-SNAPSHOT</azure-core.version>
+    <azure-core-auth.version>1.0.0-SNAPSHOT</azure-core-auth.version>
+    <azure-core-mgmt.version>1.0.0-SNAPSHOT</azure-core-mgmt.version>
```

Comment:
I thought the rename to management was also part of this PR? Or is that in another?

Reply:
Will do in next commit. Thanks for reminding!